components/formats-bsd/src/loci/formats/out/PyramidOMETiffWriter.java

```diff
@@ -167,8 +167,6 @@ public void close() throws IOException {
           saver.overwriteIFDOffset(in, allOffsets[mainIFDIndex], nextPointer);
           saver.overwriteIFDValue(in, currentFullResolution, IFD.SUB_IFD, subIFDOffsets);
           saver.close();
-          out.close();
-          in.close();
```

Comment:
Is it correct to remove these 2 close statments?

Reply:
yes since they are closed in the ``finally``

Reply:
@jburel, except the `in` and `out` variable get overwritten on lines 158 and 153, respectively, meaning only the _last_ file in the loop will be closed.

Reply:
correct too much cleaning. I will fix that